### DISCUSSION TITLE
"by ~54%" should be "by ~46%"

### DISCUSSION THREAD

**Discussion by Gsiete** [npm/feedback#202] [Website] [answered]
In this blog post:
https://github.blog/2021-02-02-npm-7-is-now-generally-available/
In the line that says

> Reducing dependencies by ~54% (67 in npm 7 and 123 in npm 6)

It should actually be **Reducing dependencies by ~46%** or **Reducing dependencies to ~54%**. I mean, If one reduces something "by" a percentage, the percentage should represent the decrease, not the new value.

**Comment by MylesBorins** [comment, marked as answer]
Thanks for pointing this out. We've updated the post
